Userland/Applications/Assistant/Providers.cpp

```diff
@@ -68,12 +69,15 @@ void AppProvider::query(DeprecatedString const& query, Function<void(NonnullRefP
     NonnullRefPtrVector<Result> results;
 
     Desktop::AppFile::for_each([&](NonnullRefPtr<Desktop::AppFile> app_file) {
-        auto match_result = fuzzy_match(query, app_file->name());
+        auto query_and_arguments = query.split_limit(' ', 2);
+        auto app_name = query_and_arguments.is_empty() ? query : query_and_arguments[0];
```

Comment:
does the following work? (i at least believe it should)
```suggestion
        auto app_name = query_and_arguments[0];
```

Reply:
This change doesn't work on its own, because query strings can be empty. 

I've pushed a change, so that queries are no longer performed against empty strings and included this suggestion